src/en/juju-upgrade.md

```diff
@@ -0,0 +1,191 @@
+Title: Upgrading Juju software
+
+
+# Upgrading Juju software
+
+At the highest level, a Juju topology can be divided into two main parts:
+
+- Juju client
+- Juju environment (possibly several)
+
+The software at each of the above are managed differently.
+
+The client software is managed by the OS package management system whereas the
+software running in an environment, including the state server (bootstrap
+node), is managed by Juju itself. This section is primarily devoted to
+describing the procedure for upgrading this environment software.
+
+
+## Upgrading the client software
+
+The client software is managed by the OS package management system. With Ubuntu
+this is APT. To upgrade the client, therefore, it is a matter of:
+
+```bash
+sudo apt-get update
+sudo apt-get install juju-core
+```
+
+## Terminology - environment software
```

Comment:
this title says 'environment software', but below you say you are going to call it "server software"

Maybe it should say "Terminology - the software running in a Juju environment" or something more explicit/descriptive